x-pack/plugins/cases/server/client/utils.ts

```diff
@@ -483,12 +544,19 @@ export async function ensureAuthorized({
   authorization: PublicMethodsOf<Authorization>;
   auditLogger?: AuditLogger;
 }) {
-  try {
-    return await authorization.ensureAuthorized(owners, operation);
-  } catch (error) {
+  const logSavedObjects = ({ outcome, error }: { outcome?: EcsEventOutcome; error?: Error }) => {
     for (const savedObjectID of savedObjectIDs) {
-      auditLogger?.log(createAuditMsg({ operation, error, savedObjectID }));
+      auditLogger?.log(createAuditMsg({ operation, outcome, error, savedObjectID }));
     }
+  };
+
+  try {
+    await authorization.ensureAuthorized(owners, operation);
+
+    // log that we're attempting an operation
+    logSavedObjects({ outcome: ECS_OUTCOMES.unknown });
```

Comment:
This utility function now handles logging after an `ensureAuthorized` call to remove duplication in each of the routes.